fetch.bs

```diff
@@ -4796,8 +4806,8 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
 
  <li><p>Let <var>response</var> be null.
 
- <li><p>Let <var>httpCache</var> be the result of <a>determining the HTTP cache partition</a>, given
- <var>httpRequest</var>.
+ <li><p>Let <var>httpCache</var> be the result of <a>determining the HTTP state partition key</a>,
+ given <var>httpRequest</var>.
```

Comment:
This returns a key, not a cache. I think that this illustrates that we still want an easy way to obtain a partitioned cache though and not inline that in various places.

Reply:
This would be just as accurate if this were null-checking checking the network partition key, and I just renamed httpCache here to networkPartitionKey - I just forgot to rename the variable.  That having been said, I've brought back the cache section, though it's pretty tiny now, and reverted this to directly reference the cache. That does seem less regression prone.